client/state/comments/actions.js

```diff
@@ -23,6 +23,8 @@ import {
 } from '../action-types';
 import { NUMBER_OF_COMMENTS_PER_FETCH } from './constants';
 
+const { isEnabled } = config;
```

Comment:
Seems like this fails the comment actions tests, we probably need to update how we mock `isEnabled` there.